src/main/java/org/apache/commons/lang3/Validate.java

```diff
@@ -156,6 +156,30 @@ public static void isTrue(final boolean expression, final String message, final
         }
     }
 
+    /**
+     * Validate that the argument condition is {@code true}; otherwise
+     * throwing an exception. This method is useful when validating according
```

Comment:
`an exception with the specified message`

Note: This change makes the code example on line 143 invalid. Please update the javadoc in the preceding method to remove:
```Validate.isTrue(myObject.isOk(), "The object is not okay");</pre>```

Reply:
Done.